zerver/tests/test_users.py

```diff
@@ -382,29 +382,28 @@ def test_access_user_by_id(self) -> None:
 
         # Must be a valid user ID in the realm
         with self.assertRaises(JsonableError):
-            access_user_by_id(iago, 1234)
+            access_user_by_id(iago, target_user_id=1234)
         with self.assertRaises(JsonableError):
-            access_user_by_id(iago, self.mit_user("sipbtest").id)
+            access_user_by_id(iago, target_user_id=self.mit_user("sipbtest").id)
 
-        # Can only access bot users if allow_deactivated is passed
+        # Can only access bot users if allow_bots is passed
         bot = self.example_user("default_bot")
-        access_user_by_id(iago, bot.id, allow_bots=True)
+        access_user_by_id(iago, target_user_id=bot.id, allow_bots=True)
         with self.assertRaises(JsonableError):
-            access_user_by_id(iago, bot.id)
+            access_user_by_id(iago, target_user_id=bot.id)
 
         # Can only access deactivated users if allow_deactivated is passed
         hamlet = self.example_user("hamlet")
         do_deactivate_user(hamlet)
         with self.assertRaises(JsonableError):
-            access_user_by_id(iago, hamlet.id)
-        access_user_by_id(iago, hamlet.id, allow_deactivated=True)
+            access_user_by_id(iago, target_user_id=hamlet.id)
+        access_user_by_id(iago, target_user_id=hamlet.id, allow_deactivated=True)
```

Comment:
Most of this code would not need to change if we didn't change `target_user_id` to be a required argument, which I don't think there's a good reason to.

Reply:
Made these suggested changes as due to change from above comment we can now pass it without using keyword notation.